client/src/helpers/combineClasses.helper.ts

```diff
@@ -0,0 +1,5 @@
+export function combineClasses(...classes: (string | undefined | null)[]): string {
+	return classes
+		.filter(className => Boolean(className))
+		.join(' ');
+}
```

Comment:
Let's use [clsx](https://www.npmjs.com/package/clsx) for such logic with classes